_config/auth.yml

```diff
@@ -1,8 +0,0 @@
----
-Name: realmeauthentication
----
-SilverStripe\Core\Injector\Injector:
-  SilverStripe\Security\Security:
-    properties:
-      Authenticators:
-        RealMe: %$SilverStripe\RealMe\Authenticator
```

Comment:
Are there any environment variables that we can check for existence of instead of skipping it entirely?

Reply:
I considered using environment variables but I thought considering the amount of configuration that's required anyway it would be more effective to opt in.